apps/fluent-tester/src/FluentTester.tsx

```diff
@@ -155,7 +156,8 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
                 onClick={() => setSelectedTestIndex(index)}
                 style={fluentTesterStyles.testListItem}
                 testID={description.testPage}
-                componentRef={index === 0 ? ref : undefined}
+                {...(index === 0 && { componentRef: win32FocusOnMount })}
+                // This ref so focus can be set on it when the app mounts in win32. Without this, focus won't be set anywhere.
```

Comment:
3 small nits:
1) Comment is usually above the line of code
2) I don't think you need the outer `{}` 
3) Prefer the name `focusOnMountRef` to `win32 focusOnMountRef` :)

Reply:
I'll fix nits 1 & 3. For nit 2, my linter goes crazy when removing the outer `{}` so that will stay.